src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactory.cs

```diff
@@ -40,7 +39,7 @@ public static AIFunction Create(Delegate method, AIFunctionFactoryCreateOptions?
     /// <param name="method">The method to be represented via the created <see cref="AIFunction"/>.</param>
     /// <param name="name">The name to use for the <see cref="AIFunction"/>.</param>
     /// <param name="description">The description to use for the <see cref="AIFunction"/>.</param>
-    /// <param name="serializerOptions">The <see cref="JsonSerializerOptions"/> used to marshal function parameters.</param>
+    /// <param name="serializerOptions">The <see cref="JsonSerializerOptions"/> used to marshal function parameters and return value.</param>
```

Comment:
Nit:
```suggestion
    /// <param name="serializerOptions">The <see cref="JsonSerializerOptions"/> used to marshal function parameters and any return value.</param>
```
or
```suggestion
    /// <param name="serializerOptions">The <see cref="JsonSerializerOptions"/> used to marshal any function parameters and return value.</param>
```
As written, though, it sounds strange to my ears without an article, e.g. "used to marshal return value".